ckanext/spatial/geoalchemy_common.py

```diff
@@ -37,7 +37,7 @@
 
 def postgis_version():
 
-    result = Session.execute('SELECT postgis_lib_version()')
+    result = Session.execute('SELECT PostGIS_Lib_Version()')
```

Comment:
that file shouldn't have been with the pull request, my bad, i will remove that commit